Client/javascripts/es6/script.js

```diff
@@ -173,6 +173,36 @@ function forgotPassword() {
 	}
 }
 
+// eslint-disable-next-line
```

Comment:
Why are you disabling eslint here?

Reply:
Eslint was saying 'viewEntries' variable has been defined but is not being used. But actually it is called from an onclick event on the browser.